docker/parachain-launch-config-dev.yml

```diff
@@ -4,14 +4,15 @@
 #
 # Relaychain Configuration
 relaychain:
-  image: parity/polkadot:v0.9.12 # the docker image to use
+  image: parity/polkadot:latest # the docker image to use
```

Comment:
Why we use latest?

Reply:
My motivation is that with this we could notice the incompatibility earlier.
Theoretically, there should be no problem connecting an older client to the latest mainnet(polkadot), but if there is, it's a good indication that we should update our client.